src/Illuminate/Http/Resources/DelegatesToResource.php

```diff
@@ -50,7 +50,7 @@ public function resolveRouteBinding($value)
      */
     public function offsetExists($offset)
     {
-        return array_key_exists($offset, $this->resource);
+        return isset($this->resource[$offset]);
```

Comment:
isset and array_key_exists are not the same behavior?

Reply:
`array_key_exists` is also faster than isset. Ref: https://github.com/php/php-src/pull/3360

Reply:
@taylorotwell The behavior is different, but isset() is "correct" where the contract of offsetExists() is concerned (it should return false if the value is null, otherwise isset() on the object will not work correctly). From that perspective I think this is a fairly safe change.

@KennedyTedesco No, array_key_exists() is not faster in the way it is used here.

Reply:
@nikic Thanks for pointing this out. 👍

Reply:
@taylorotwell I think @nikic explained this well :)